### ISSUE TITLE
Issue setting up Velero 1.3.0 on AKS

### ISSUE THREAD

**Issue by skri547** [vmware-tanzu/velero#4534]
Hi Team, I have used the below command to install the velero on AKS Cluster

I am using the setup with Service principal method which has access to the Resource Group, and installed velero with the below command by following this doc - https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure
velero install
--provider azure
--plugins velero/velero-plugin-for-microsoft-azure:v1.3.0
--bucket $STORAGE_CONTAINER_NAME
--secret-file ./credentials-velero
--backup-location-config resourceGroup=$AZURE_RESOURCE_GROUP,storageAccount=$STORAGE_ACCOUNT,subscriptionId=$AZURE_SUBSCRIPTION_ID
--snapshot-location-config resourceGroup=$AZURE_RESOURCE_GROUP,subscriptionId=$AZURE_SUBSCRIPTION_ID

And All the resources got deployment and I see some errors in the pods logs when checked, attached below
velero

I am trying to exec in to pod but its not letting me in.
image

If you could please provide me the solution to the problem that would be great. Let me know if you want to share any other details

**Comment by ywk253100** [comment]
There are two similar resource groups that are confused in the doc: `AZURE_RESOURCE_GROUP ` and `AZURE_BACKUP_RESOURCE_GROUP `. Seems you mix these two RGs up

**Comment by skri547** [comment]
Thank you so much @ywk253100 , can you please link to the right document which can help me on doing some backup and restore. I couldn't find the document that has these details.

**Comment by ywk253100** [comment]
Your referenced doc is the correct one.
The resource group used in `--backup-location-config` should be `$AZURE_BACKUP_RESOURCE_GROUP` rather than `$AZURE_RESOURCE_GROUP`. 
Please double-check other related configurations

**Comment by skri547** [comment]
Yeah @ywk253100  I understood it, After you said I tried the implementation and it worked, but I have a one last question.

I did the back of the cluster A and i want to restore the backup of Cluster A in Cluster B But how do i restore in Cluster B? Does that have info when i install velero on Cluster B using same storage account which was used in cluster A? Can you please give me the details for this process on how restore can be done and also would be great if you include a document of the similar process.

Thanks in Advance

**Comment by ywk253100** [comment]
Please refer to https://velero.io/docs/v1.8/migration-case/#docs

**Comment by skri547** [comment]
@ywk253100  So, I have a question now, If we do a backup on the cluster which has its own cluster related variables or values like managedID  or resource group name. If we do the restore, how would that work? it will throw error right? when the restore operation is performed, its wont see those becuause this setup like DR might be present in the different RG and different region. How could we solve these problems?

**Comment by skri547** [comment]
@ywk253100  any update on the this?

**Comment by ywk253100** [comment]
Velero cannot backup in a region and do restore in another region on Azure. 
For backup/restore across different subscription/resource-group please refer to this [commet](https://github.com/vmware-tanzu/velero/issues/4191#issuecomment-1085378429).

I'm closing this issue as the original issue is resolved and if you have any other questions about the across subscription restoring please comment in #4191
